### ISSUE TITLE
Add "Note:" for `cast_target_dtype_to: "$torch.float32"` in Task 2 and 3 `foundation_finetune.yaml`

### ISSUE THREAD

**Issue by ibro45** [AIM-Harvard/foundation-cancer-image-biomarker#24]
The training crashes without casting. I have seen that it's correctly present in `supervised_finetune.yaml`. However, check `post_criterion_activation` in `supervised_finetune.yaml`, I think that should be `softmax` for task 1.
